.idea/inspectionProfiles/Project_Default__with_Android_.xml

```diff
@@ -0,0 +1,31 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Delete this also

Reply:
[Done](https://github.com/kiwix/kiwix-android/pull/1276/commits/35a0bffb478e279758c4096d8ced4ea571e3771e)